### PR TITLE
update(userspace): small improvements

### PR DESCRIPTION
**Pull request by Andreagit97**
**What type of PR is this?**

/kind cleanup

**Any specific area of the project related to this PR?**

/area libscap

/area tests

**Does this PR require a change in the driver versions?**

No

**What this PR does / why we need it**:

This PR brings really small improvements :) 

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -4,7 +4,7 @@ BasedOnStyle: LLVM
 AccessModifierOffset: -8
 BreakBeforeBraces: Allman
 BreakConstructorInitializers: AfterColon
-ColumnLimit: 0
+ColumnLimit: 120
```
Comment (Andreagit97) [inline .clang-format:7]: this has no effect on our code, it seems a more reasonable value for developers that use it locally

File: userspace/libscap/linux/scap_procs.c [hunk 2]
```diff
@@ -1355,7 +1359,7 @@ struct scap_threadinfo* scap_proc_get(scap_t* handle, int64_t tid, bool scan_soc
 	//
 	// No /proc parsing for offline captures
 	//
-	if(handle->m_mode == SCAP_MODE_CAPTURE)
+	if(handle->m_mode == SCAP_MODE_CAPTURE || handle->m_mode == SCAP_MODE_TEST)
```
Comment (Andreagit97) [inline userspace/libscap/linux/scap_procs.c:1362]: no reason to have proc scan enabled in test mode

**Review by FedeDP — APPROVED** [review]
/approve

**Review by LucaGuerra — APPROVED** [review]
LGTM

**Review by jasondellaluce — APPROVED** [review]

**Pull request merged**